src/components/ConversationSettings/LockingSettings.vue

```diff
@@ -68,9 +85,16 @@ export default {
 		},
 	},
 
+	mounted() {
+		this.isReadOnlyStateLoading = true
+		const conversation = this.$store.getters.conversation(this.token)
```

Comment:
this needs to go into a computed otherwise it will never update when someone else changes the state?